src/search&currentWeather/CurrentWeatherAside.jsx

```diff
@@ -0,0 +1,39 @@
+const CurrentWeatherAside = ({ currentWeather, units }) => {
+  return (
+    <div>
+      <div className="weather-image-container">
+        <img
+          className="weather-image"
+          src={` https://openweathermap.org/img/wn/${currentWeather.weather[0].icon}@4x.png`}
```

Comment:
This logic of creating src is duplicated
consider creating a separate component, for example name it `WeatherImage`

encapsulate logic inside it